www/docs/en/dev/config_ref/index.md

```diff
@@ -45,207 +45,356 @@ passively copied into various `platforms/` subdirectories, for example:
+   versionCode (string) | *Required: false* <br/> **Platforms supported: Android** <br/> Alternative version for Android.
+   CFBundleVersion (string) | *Required: false* <br/> **Platforms supported: iOS** <br/> Alternative version for iOS.  
+   packageVersion (string) | *Required: false* <br/> **Platforms supported: Windows** <br/> Alternative version for Windows.
+   packageName (string) | *Default: Cordova.Example* <br/> **Platforms supported: Windows** <br/> Package name for Windows.
+   xmlns (string) | *Required: true* <br/> namespace for the config.xml document.
+   xmlns:cdv (string) | *Required: true* <br/> namespace prefix.
+    
+   Examples:
 
-Android, iOS, and Windows support a second version string (or number) in 
-addition to the one visible in app stores:
+    <widget id="io.cordova.hellocordova">
+    </widget>
 
-- [versionCode](http://developer.android.com/tools/publishing/versioning.html)
-  for Android
-- [CFBundleVersion](http://stackoverflow.com/questions/4933093/cfbundleversion-in-the-info-plist-upload-error)
-  for iOS
-- [packageVersion](https://msdn.microsoft.com/en-us/library/windows/apps/br211441.aspx)
-  for Windows
+    <widget version="0.0.1">
+    </widget>
 
-Below is an example that explicitly sets these properties:
+    <!-- Android -->
+    <widget versionCode="0.1.3">
+    </widget>
 
-        <widget id="io.cordova.hellocordova"
-          version="0.0.1"
-          android-versionCode="7"
-          ios-CFBundleVersion="3.3.3"
-          windows-packageVersion="1.0.1.2">
+    <!-- iOS -->
+    <widget CFBundleVersion="0.1.3">
+    </widget>
 
-If alternative version is not specified, the following
-defaults will be used:
+    <!-- Windows -->
+    <widget packageVersion="0.1.3">
+    </widget>
 
-        // assuming version = MAJOR.MINOR.PATCH-whatever
-        versionCode = PATCH + MINOR * 100 + MAJOR * 10000
-        CFBundleVersion = "MAJOR.MINOR.PATCH"
-        packageVersion = "MAJOR.MINOR.PATCH.0"
+    <widget xmlns="http://www.w3.org/ns/widgets">
+    </widget>
 
-## Global Preferences
 
-The following global preferences apply to all platforms:
+## name
+   Specifies the app's formal name, as it appears on the device's home screen and within app-store interfaces.
 
-- `Fullscreen` allows you to hide the status bar at the top of the
-  screen. The default value is `false`. Example:
+   Example:
 
-        <preference name="Fullscreen" value="true" />
+    <widget ...>
+        <name>HelloCordova</name>
+    </widget>
+    
+## description
+   Specifies metadata that may appear within app-store listings.
 
-## Multi-Platform Preferences
+   Example:
+   
+    <widget ...>
+        <description>A sample Apache Cordova application</description>
+    </widget>
 
-The following preferences apply to more than one platform, but not to
-all of them:
+    
+## author
+   Specifies contact information that may appear within app-store lisitngs.
+   
+   Attributes (type) | Description
```

Comment:
For consistency, and according to the style guide, there's no space between "Attributes" and "(type)".